app/controllers/claims_controller.rb

```diff
@@ -5,7 +5,9 @@ def index
     # TODO: delete old claims without a claim type or avoid creating
     # claim before we have a claim type - this breaks the pattern we
     # have used for the forms.
-    filtered_claims = Claim.where(claim_type: ClaimType::SUPPORTED.map(&:to_s)).order('updated_at DESC')
+    # debugger
+    # filtered_claims = Claim.where(claim_type: ClaimType::SUPPORTED.map(&:to_s)).order('updated_at DESC')
+    filtered_claims = Claim.all.order('updated_at DESC')
```

Comment:
couple of things here:
* don't leave comments in
* the `all` call is not required, it could just be `Claim.all.order('updated_at DESC')`
* this would allow some bad data to sneak in we should we use `Claim.where.not(ufn: nil).order('updated_at DESC')`

Reply:
getting: 
Failures:

  1) ClaimsController index renders successfully with claims
     Failure/Error: filtered_claims = Claim.where.not(ufn: nil).order('updated_at DESC')
       #<Double (anonymous)> received unexpected message :not with ({:ufn=>nil})
     # ./app/controllers/claims_controller.rb:8:in `index'
     # ./spec/controllers/claims_controller_spec.rb:10:in `block (3 levels) in <top (required)>'

Reply:
this is for the rspec

Reply:
this is for the rspec